### PR TITLE
reference-manual: boards: update images

### PR DESCRIPTION
**Pull request by munoz0raul**
Some images were out of date and tried to align with the flash instructions from the GS.

Signed-off-by: Raul Muñoz <raul@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 16072bed1f5c7dffd28eb58d76f3c5ec9ad8ed85 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1010/docs/artifacts/html/index.html

**Comment by mwasilew** [comment]
LGTM

File: source/reference-manual/boards/apalis-imx6-prepare.rst [hunk 1]
```diff
@@ -22,7 +22,8 @@ Download necessary files from ``https://app.foundries.io/factories/<factory>/tar
 
    .. figure:: /_static/boards/apalis-imx6-steps-2.png
       :align: center
-      :width: 400
+      :width: 450
```
Comment (vanmaegima) [inline source/reference-manual/boards/apalis-imx6-prepare.rst:25]: Shouldn't we update this image and artifact lists as well? I believe apalis-imx6 will generate the sit file in the latest builds.

File: source/reference-manual/boards/imx8-prepare.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ Download necessary files from ``https://app.foundries.io/factories/<factory/targ
 
    .. figure:: /_static/boards/imx-steps-2.png
       :align: center
-      :width: 300
+      :width: 769
```
Comment (vanmaegima) [inline source/reference-manual/boards/imx8-prepare.rst:27]: imx-steps-2 image also doesn't include the sit.bin

**Comment by doanac** [comment]
Docs for 5bd1b92ff93ec27ec4b2200aa54d419201977c41 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1012/docs/artifacts/html/index.html

File: source/reference-manual/boards/apalis-imx6-prepare.rst [hunk 3]
```diff
@@ -4,25 +4,28 @@ Preparation
...
+        with the name of the board and **download the Factory image for that
+        machine.**
 
-   | E.g: ``lmp-factory-image-apalis-imx6.wic.gz``
-   |      ``SPL-apalis-imx6``
-   |      ``u-boot-apalis-imx6.itb``
+        | E.g: 
+        |     ``lmp-factory-image-<machine_name>.wic.gz``
+        |     ``u-boot-<machine-name>.itb``
+        |     ``sit-<machine-name>.bin``
```
Comment (igoropaniuk) [inline source/reference-manual/boards/apalis-imx6-prepare.rst:22]: sometimes you use machine_name with underscore, sometimes it's with hyphen like here

File: source/reference-manual/boards/imx8-prepare.rst [hunk 4]
```diff
@@ -43,10 +45,11 @@ Download necessary files from ``https://app.foundries.io/factories/<factory/targ
 
 #. Organize all the files like the tree below::
 
-      ├── imx-boot-<machine_name>
-      ├── u-boot-<machine.name>.itb             # imx8mm only
-      ├── sit-<machine-name>.bin                # imx8mm only
       ├── lmp-factory-image-<machine_name>.wic
+      ├── u-boot-<machine.name>.itb
```
Comment (igoropaniuk) [inline source/reference-manual/boards/imx8-prepare.rst:49]: here dot  is used :)

**Comment by doanac** [comment]
Docs for 27c935dab017c49fd25018058232e98d73106d5c are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1014/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for cc2b49bf8e8e2bb545c62075b0cf1a681eebcd9e are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1015/docs/artifacts/html/index.html

File: source/reference-manual/boards/apalis-imx6-prepare.rst [hunk 5]
```diff
@@ -39,13 +42,14 @@ Download necessary files from ``https://app.foundries.io/factories/<factory>/tar
...
-      │   ├── u-boot-mfgtool.itb
-      │   ├── uuu
-      │   └── uuu.exe
-      ├── SPL-apalis-imx6
-      └── u-boot-apalis-imx6.itb
+      ├── lmp-factory-image-<machine-name>.wic.gz
+      ├── u-boot-<machine-name>.itb
+      ├── sit-<machine-name>.bin
+      ├── SPL-<machine-name>
+      └── mfgtool-files-imx6ullevk
```
Comment (angolini) [inline source/reference-manual/boards/apalis-imx6-prepare.rst:49]: I think this `imx6ullevk` is not intended here

File: source/reference-manual/boards/imx6-prepare.rst [hunk 6]
```diff
@@ -40,13 +43,14 @@ Download necessary files from ``https://app.foundries.io/factories/<factory>/tar
...
-      │   ├── u-boot-mfgtool.itb
-      │   ├── uuu
-      │   └── uuu.exe
-      ├── SPL-imx6ullevk
-      └── u-boot-imx6ullevk.itb
+      ├── lmp-factory-image-<machine-name>.wic.gz
+      ├── u-boot-<machine-name>.itb
+      ├── sit-<machine-name>.bin
+      ├── SPL-<machine-name>
+      └── mfgtool-files-imx6ullevk
```
Comment (angolini) [inline source/reference-manual/boards/imx6-prepare.rst:50]: It this it's not intended here

File: source/reference-manual/boards/imx8-prepare.rst [hunk 7]
```diff
@@ -4,50 +4,53 @@ Preparation
...
 
-      gunzip lmp-factory-image-<machine_name>.wic.gz
+      gunzip lmp-factory-image-<machine-name>.wic.gz
 
 #. Expand the **run** in the :guilabel:`Runs` section which corresponds
    with the name of the board mfgtool-files and **download the tools for that
    machine.**
 
-   E.g: ``<machine_name>-mfgtools``
+   E.g: ``<machine-name>-mfgtools``
```
Comment (angolini) [inline source/reference-manual/boards/imx8-prepare.rst:39]: I think the file name here is inverted

**Review by angolini — APPROVED** [review]

File: source/reference-manual/boards/apalis-imx6-prepare.rst (hunk 3 above)
Comment (vanmaegima) [reply source/reference-manual/boards/apalis-imx6-prepare.rst:22]: @munoz0raul I believe these are for you :)

**Comment by doanac** [comment]
Docs for af928bfd0f7848c47f2ec01bfbab78182047a681 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1016/docs/artifacts/html/index.html

**Review by Tim-Anderson — APPROVED** [review]
not sure why you did the substitute <machine-name> on specific platforms. I understand it for generic platforms like raspberrypi otherwise LGTM

**Pull request merged**